views/apply.njk

```diff
@@ -0,0 +1,35 @@
+{% extends "./layouts/main.njk" %}
+
+{% block pageTitle %}
+  Apply for Role
+{% endblock %}
+
+{% block links %}
+<link rel="stylesheet" href="/css/style.css">
+{% endblock %}
+
+{% block content %}
+
+
+<div class="d-flex justify-content-center "">
+  <div>
```

Comment:
Unnecessary div

Reply:
When I removed this div, it uncenters all of the content
<img width="1412" alt="Screenshot 2024-09-11 at 13 25 50" src="https://github.com/user-attachments/assets/e34cfa88-1d99-444e-b30e-5f4d74b6c79c">